package.json

```diff
@@ -26,6 +26,7 @@
     "express": "^4.17.1",
     "express-handlebars": "^4.0.3",
     "express-session": "^1.17.2",
+    "expression": "^0.0.1",
```

Comment:
Do we need this installed?  I don't see it being imported anywhere, if not just make a commit after you `npm un expression`

Reply:
Not sure when & why I installed that. I just uninstalled it and committed the change. Thanks!